sigle/src/modules/layout/components/AppHeader.tsx

```diff
@@ -61,6 +63,17 @@ const StatusDot = styled('div', {
 });
 
 export const AppHeader = () => {
+  const { data: settingsFile } = useQuery(
```

Comment:
wdyt about moving this to the new "hooks" folder, we could create an `appData` folder and put all the settings, following etc inside

Reply:
yeah that makes sense :)